FV3/atmos_cubed_sphere/tools/fv_diagnostics.F90

```diff
@@ -1132,6 +1141,15 @@ subroutine fv_diag_init(Atm, axes, Time, npx, npy, npz, p_ref)
         Atm(n)%nudge_diag%nudge_u_dt(isc:iec,jsc:jec,npz) = 0.0
     endif
 
+    idiag%id_column_eastward_acceleration_nudge = register_diag_field('dynamics', &
+          'column_eastward_acceleration_nudge', axes(1:2), Time, &
+          'column integrated eastward wind tendency from nudging', &
+          'Pa', missing_value=missing_value)
```

Comment:
column integrated wind tendency is in pressure units?

Reply:
Yeah, the column integral is $1/g \int du/dt dp$ so you end up with pressure as the units, which is the unit used for wind stress.

Reply:
this is the typical unit to report column-integrated momentum tendencies, e.g. due to gravity wave drag.